.gitignore

```diff
@@ -11,3 +11,10 @@
 /output/*/graphs
 /output/*/logging.txt
 /__pycache__
+
+build/
+causing/__pycache__/
```

Comment:
We could just add `__pycache__` (without preceding slash) once. I'm sure we want to add any `__pycache__` directories.

Reply:
okay